components/collector/tests/source_collectors/harbor_json/test_source_up_to_dateness.py

```diff
@@ -0,0 +1,17 @@
+"""Unit tests for the Harbor JSON source up-to-dateness collector."""
+
+from collector_utilities.date_time import days_ago, parse_datetime
+
+from .base import HarborJSONCollectorTestCase
+
+
+class HarborJSONSourceUpToDatenessTest(HarborJSONCollectorTestCase):
+    """Unit tests for the source up-to-dateness metric."""
+
+    METRIC_TYPE = "source_up_to_dateness"
+
+    async def test_souce_up_to_dateness(self):
+        """Test the source up-to-dateness."""
+        response = await self.collect(get_request_json_return_value=self.VULNERABILITIES_JSON)
+        expected_value = str(days_ago(parse_datetime("2023-08-26T16:32:21.923910328Z")))
```

Comment:
Shouldn't we declare the `datetime.datetime` instead of re-using `parse_datetime`, or can we assume that `parse_datetime` is sufficiently tested elsewhere?